fastlane/download_metadata.swift

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env swift
+
+import Foundation
+
+let glotPressSubtitleKey = "app_store_subtitle"
+let glotPressWhatsNewKey = "v1.12-whats-new"
+let glotPressDescriptionKey = "app_store_desc"
+let glotPressKeywordsKey = "app_store_keywords"
+let baseFolder = "./metadata"
+
+// iTunes Connect language code: GlotPress code
+let languages = [
+    "de-DE": "de",
+    "default": "en-us", // Technically not a real GlotPress language
+    "en-US": "en-us", // Technically not a real GlotPress language
+    "es-ES": "es",
+    "fr-FR": "fr",
+    "id": "id",
+    "it": "it",
+    "ja": "ja",
+    "ko": "ko",
+    "nl-NL": "nl",
+    "pt-BR": "pt-br",
+    "ru": "ru",
+    "sv": "sv",
+    "tr": "tr",
+    "zh-Hans": "zh-cn",
+    "zh-Hant": "zh-tw",
+]
+
+func downloadTranslation(languageCode: String, folderName: String) {
+    let languageCodeOverride = languageCode == "en-us" ? "es" : languageCode
+    let glotPressURL = "https://translate.wordpress.com/projects/simplenote%2Fmacos%2Frelease-notes/\(languageCodeOverride)/default/export-translations?format=json"
+    let requestURL: URL = URL(string: glotPressURL)!
+    let urlRequest: URLRequest = URLRequest(url: requestURL)
+    let session = URLSession.shared
+
+    let sema = DispatchSemaphore( value: 0)
+
+    print("Downloading Language: \(languageCode)")
+	
+    let task = session.dataTask(with: urlRequest) {
+        (data, response, error) -> Void in
+
+        defer {
+            sema.signal()
+        }
+
+        guard let data = data else {
+            print("  Invalid data downloaded.")
+            return
+        }
+
+        guard let json = try? JSONSerialization.jsonObject(with: data, options: []),
+            let jsonDict = json as? [String: Any] else {
+                print("  JSON was not returned")
+                return
+        }
+
+        var subtitle: String?
+        var whatsNew: String?
+        var keywords: String?
+        var storeDescription: String?
+
+        jsonDict.forEach({ (key: String, value: Any) in
+
+            guard let index = key.firstIndex(of: Character(UnicodeScalar(0004))) else {
```

Comment:
Nice touch upgrading this to use `firstIndex(of:)` instead of the deprecated `index(of:)`.

Reply:
Thank you! 😃